molgenis/molgenis-emx2
# fix: Load images from application jar

What are the main changes you did:
- Use filesystem path walker in case of file in jar 

Closes #3512

how to test:
- load profile with data ( and files ) contained in jar , images should now be loaded from jar 

Use the 'IMAGE_TEST' profile and include the demodata to test with minimal example, this profile can be removed after initial review and we can extend the catalogue example to include images

See: https://github.com/molgenis/molgenis-emx2/pull/4393 for fix ( excludes 'IMAGE_TEST' from options list in create modal)

todo:
